README.md

```diff
@@ -21,12 +21,18 @@
 
 ```bash
 yarn install
-NODE_ENV=development node_modules/.bin/webpack-cli --mode development
+yarn build
 ```
 #### Enable debug mode
 - `touch local_settings.py`
 - Inside `local_setting.py`, have `DEBUG = True`
 
+If you experience missing locale, try add locale to your shell script
```

Comment:
If you experience a missing locale error, you can try configuring the locale:
```
<commands here>
```

Reply:
Fixed in new commit.